treeherder/model/migrations/0055_autoclassify_status.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0054_duplicate_failure_classifications'),
+    ]
+
+    operations = [migrations.RunSQL(
+        sql="""
+BEGIN;
+--
+-- Add field autoclassify_status to job
+--
+SET FOREIGN_KEY_CHECKS=0;
+ALTER TABLE `job` ADD COLUMN `autoclassify_status` integer DEFAULT 0 NOT NULL;
+ALTER TABLE `job` ALTER COLUMN `autoclassify_status` DROP DEFAULT;
+COMMIT;
```

Comment:
I am curious if the manual SQL for this migration has any advantages over the automatically generated sql by Django? I know (boy do I know) that sometimes Django can be inefficient at doing these things, but I don't see how that would be the case here.

I'd prefer to use the default behaviour where possible, as it would make an eventual migration to another database easier (at the very least there would be less to audit).